src/sparkx/flow/LeeYangZeroFlow.py

```diff
@@ -0,0 +1,559 @@
+from sparkx.flow import FlowInterface
+import numpy as np
+import random as rd
+import warnings
+
+rd.seed(42)
```

Comment:
Why do we fix the random seed?

Reply:
I left this in to have reproducible results. Do you think we should change this or let the user give the seed?

Reply:
So it is basically only used for the uniform distribution. I think it should be fine to keep it like this.